frontend/README.md

```
@@ -1,3 +1,165 @@
+
+### 3. Android Studioで開く
+
+```bash
+npm run android:open
+```
+
+Android Studioで実機を選択して実行してください。
+
+## プロジェクト構造
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)